lib/ret/storage.ex

```diff
@@ -83,6 +83,7 @@ defmodule Ret.Storage do
     |> Enum.map(fn
       {k, {id, key}} -> {k, promote(id, key, nil, account)}
       {k, {id, key, promotion_token}} -> {k, promote(id, key, promotion_token, account)}
+      {k, :remvoe} -> {k, {:ok, :remove}}
```

Comment:
typo. this probably deserves an integration test :)

Reply:
woops, changed that right before commit, but ended up moving this elsewhere anyway.